### DISCUSSION TITLE
240327

### DISCUSSION THREAD

**Discussion by jisunipark** [codeit-part4-8team-project/main#83] [팀회의]

**Comment by sobokki** [comment]
## 논의사항
## 어제 한 일
그룹필터 오류 수정했습니다(많이 못했어요 ㅜ)
## 오늘 할 일
다시 그룹필터 수정하고 데이터 받아오는 구조 수정해야 할 것 같습니다
## 오늘의 일정 (코어 타임)
## 멘토님 질문 사항

**Comment by jisunipark** [comment]
## 논의사항
- 필겸님 리뷰 확인했습니다. useParams 쓰면 teamId를 받는 방법이 조금 달라질 것도 같은데 일단 해봐야 알 것 같아요! (useParams 사용할지 useTeam 사용할지)

## 어제 한 일
- 자유게시판/이슈/공지사항 api 반영 + 관심사 분리
- 자유게시판/이슈/공지사항 ui 수정

## 오늘 할 일
- 공지사항 부분 마무리
- 자유게시판 페이지네이션

## 오늘의 일정 (코어 타임)

## 멘토님 질문 사항

**Comment by caseV12** [comment]
## 논의사항

## 어제 한 일
![스크린샷 2024-03-27 051838](https://github.com/codeit-part4-8team-project/main/assets/121015462/4d547c6b-587f-41c8-b955-996cb0683030)
userContext를 두 컴포넌트에서 호출한다면... 해결 결국 못해서 질의응답 가려합니다

members 건드리기 시작
## 오늘 할 일
members 끝내기, 저거 어떻게 해보기
## 오늘의 일정 (코어 타임)
x
## 멘토님 질문 사항

**Comment by MoonPillGyeom** [comment]
## 논의사항
x
## 어제 한 일
각 모달 리팩터링을 좀 했습니다
+ 멤버리스트 초대부분을 아직 구현을 못했는데 헷갈려서 나중에 여쭤볼려고합니다
## 오늘 할 일
멤버 리스트, 에러메세지만들기
## 오늘의 일정 (코어 타임)
x
## 멘토님 질문 사항
x
